modules/seed/addons/monitoring/templates/manifests/grafana.yaml

```diff
@@ -113,7 +113,7 @@ spec:
       #  command: ['sh', '-c', 'until wget -T 5 -qO- http://prometheus-web:8080/status > /dev/null; do echo waiting for Prometheus; sleep 2; done;']
       containers:
       - name: grafana
-        image: grafana/grafana:4.6.3
+        image: grafana/grafana:5.0.0-beta1
```

Comment:
Would you really use a beta version of Grafana instead of a stable one?

Reply:
Sure thing. I didn't see any showstoppers for the 5.x beta. It will be GA anytime soon.

Reply:
so if you doesn't see any concern, then lgtm.

Reply:
@dkistner can you approve this PR